manager/integration/tests/test_migration.py

```diff
@@ -147,7 +146,55 @@ def test_migration_with_failed_replica(clients, volume_name):  # NOQA
        And the old failed replica will be cleaned up.
     8. Validate initially written test data.
     """
-    return
+    def finalizer():
+        exec_cmd = ["mkdir", "/var/lib/longhorn/replicas"]
```

Comment:
Use `["mkdir", "-p", "/var/lib/longhorn/replicas"]` to avoid the directory-already-exist error

Reply:
Hi @shuo-wu .

I modified the code, thank you